.cargo/config.toml

```diff
@@ -28,7 +28,6 @@ rustflags = [
     "-Wclippy::float_cmp_const",
     "-Wclippy::fn_params_excessive_bools",
     "-Wclippy::from_iter_instead_of_collect",
-    "-Wclippy::if-not-else",
```

Comment:
I removed this lint as Foundry code has a pattern of checking for error conditions first with `if !invariant { error } else { ... }`. I think this is a reasonable pattern that we shouldn't prevent.

Reply:
Why not just allow the pattern in the foundry crates?

Reply:
We're planning to heavily modify the foundry crates, and it's not great if there are different coding standards for different parts of the code base + I don't think the pattern is actually harmful.